draft-ietf-jose-fully-specified-algorithms.xml

```diff
@@ -589,6 +591,28 @@
       </t>
     </section>
 
+    <section anchor="MultiAlgs" title="Fully-Spedified Computations Using Multiple Algorithms">
+      <t>
+	Both JOSE and COSE have operations that take multiple algorithms as parameters.
+	Encrypted objects in JOSE <xref target="RFC7516"/> use two algorithm identifiers:
+	the first in the <spanx style="verb">alg</spanx> (Algorithm) Header Parameter,
+	which specifies how to determine the content encryption key, and
+	the second in the <spanx style="verb">enc</spanx> (Encryption Algorithm)
+	Header Parameter, which specifies the content encryption algorithm.
+	Likewise, encrypted COSE objects can use multiple algorithms
+	for corresponding purposes.
+      </t>
+      <t>
+	For purposes of being fully specified,
```

Comment:
```suggestion
	To be fully specified,
```

Reply:
Not sure this is any better than what you have already.

Reply:
I like my wording better, in part, because yours seems possibly a little redundant in the full context of the sentence:
> To fully specified,	each of these multiple algorithms must be independently fully specified.